pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/prerequisites/index.md

```diff
@@ -20,5 +20,29 @@ Before you begin using Konvoy, you must have:
 
 When air-gapped, you must follow the steps described in the [air-gapped prerequisites page][prerequisites-airgapped], otherwise [begin creating the bootstrap cluster][bootstrap].
 
+## Control plane nodes
+
+You should have at least three control plane nodes.
+
+Each control plane node should have at least:
+
+- 4 cores
+- 16 GiB memory
+- Approximately 80 GiB of free space for the volume used for `/var/lib/kubelet` and `/var/lib/containerd`.
+- Disk usage must be below 85% on the root volume.
+
+## Worker nodes
+
+You should have at least four worker nodes. The specific number of worker nodes required for your environment can vary depending on the cluster workload and size of the nodes.
+
+Each worker node should have at least:
+
+- 8 cores
+- 32 GiB memory
+- Approximately 80 GiB of free space for the volume used for `/var/lib/kubelet` and `/var/lib/containerd`.
+- Disk usage must be below 85% on the root volume.
+
+If you plan to use local volume provisioning to provide persistent volumes for the workloads, you must mount at least four volumes to the `/mnt/disks/` mount point on each node. Each volume must have at least 55 GiB of capacity if you are using default addon configurations.
```

Comment:
This is right, right? Addons is right here, because we're talking about the eventuality of getting from 1.x to 2?

Reply:
maybe we can get rid of this bit. its basically saying if you want to apply the kommander apps to it

Reply:
I don't think we used addon in 1.8 either. Gonna change this.